File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRepresentationItem/IfcStyledItem/IfcStyledItem.cs
public class IfcStyledItem : IfcRepresentationItem
{
    public IfcRepresentationItem Item { get; set; }
    public SET [1:?] OF IfcPresentationStyle Styles { get; set; }
    public IfcLabel Name { get; set; }

    // === WHERE CLAUSES ===
    // ApplicableItem : NOT('IFC4X3_DEV_73740fe4.IFCSTYLEDITEM' IN TYPEOF(Item))
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRepresentationItem/IfcStyledItem/IfcStyledItem.vb
Public Class IfcStyledItem Inherits IfcRepresentationItem
    Public Item As IfcRepresentationItem
    Public Styles As SET [1:?] OF IfcPresentationStyle
    Public Name As IfcLabel

    ' === WHERE CLAUSES ===
    ' ApplicableItem : NOT('IFC4X3_DEV_73740fe4.IFCSTYLEDITEM' IN TYPEOF(Item))
End Class
